insights/config/specs.py

```diff
@@ -425,6 +426,7 @@
 
 pre_commands = {
     "block"                     : "/bin/ls /sys/block | awk '!/^ram|^\\.+$/ {print \"/dev/\" $1 \" unit s print\"}'",
+    "catalina"                  : r"""/bin/ps auxww | awk '/java/ { match($0, "\\-Dcatalina\\.home=([^[:space:]]+)", a); match($0, "\\-Dcatalina\\.base=([^[:space:]]+)", b); if (a[1] != "" || b[1] != "") print a[1] " " b[1] }'""",
```

Comment:
There is not anything obvious that I think would break this, but until someone actually runs it...

Reply:
@jsvob I ran this in Bash. Could not find how it works in the Insights yet.